package.json

```diff
@@ -15,7 +15,8 @@
     "morgan": "^1.5.3",
     "node-uuid": "^1.4.3",
     "octonode": "^0.7.1",
-    "session-file-store": "0.2.0"
+    "session-file-store": "0.2.0",
+    "multer": "1.1.0"
```

Comment:
Do not add manually the multer here. The lib was not downloading automatically and npm should take care where and how to register multer as depedency.
Rollback modification and run

```
npm install multer --save
```